pkg/actions/cluster/owned.go

```diff
@@ -54,3 +60,68 @@ func (c *OwnedCluster) Upgrade(dryRun bool) error {
 	cmdutils.LogPlanModeWarning(dryRun && (stackUpdateRequired || versionUpdateRequired))
 	return nil
 }
+
+func (c *OwnedCluster) Delete(waitTimeout time.Duration, wait bool) error {
+	var (
+		clientSet kubernetes.Interface
+		oidc      *iamoidc.OpenIDConnectManager
+		err       error
```

Comment:
Minor nit: this can be declared in the `if clusterOperable {` block to limit its scope.

Reply:
`clientSet` and `oidc` get referenced outside of the `if clusterOperable` block

Reply:
I meant the `err` variable.

Reply:
ah gotcha 👍 fixed